src/python/WMCore/MicroService/MSRuleCleaner/MSRuleCleaner.py

```diff
@@ -404,6 +416,37 @@ def setPlineMarker(self, wflow, pName):
                 wflow['CleanupStatus'][pName] = False
         return wflow
 
+    def _getStatusTransitionTime(self, wflow):
```

Comment:
By the way, I agree that this should be renamed to something like `_getLastStatusTransitionTime`

Reply:
done